Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Button.tsx

```diff
@@ -14,13 +14,14 @@ interface IButtonProps extends IHaveOptionalChildrenProps, IHaveOptionalClassNam
     size?: 'small' | 'medium' | 'large',
 }
 
-interface ILinkButtonProps extends IHaveOptionalClassName {
+interface ILinkButtonProps extends IHaveOptionalClassName, IHaveOptionalChildrenProps {
     text: string;
     id?: string;
     type?: 'primary' | 'secondary' | 'plain' | 'disabled' | 'link';
     size?: 'small' | 'medium' | 'large';
     to: string;
     isToExternal?: boolean,
+    onClick?: React.MouseEventHandler<HTMLAnchorElement>,
```

Comment:
`LinkButton` should not have `onClick`. It redirects. We have `Button` for clicking.